include/ledstripgfx.h

```diff
@@ -89,8 +89,6 @@ class LEDStripGFX : public GFXBase
         #ifdef POWER_LIMIT_MW
             set_max_power_in_milliwatts(POWER_LIMIT_MW);                // Set brightness limit
         #endif
-
-        g_Values.Brightness = 255;
```

Comment:
Removed because LEDStripGFX has no business deciding what the global brightness value should be.